src/formBuilder/Card.test.js

```diff
@@ -222,5 +226,28 @@ describe('Card', () => {
 
     const inputTypeLabel = getHeadingText(wrapper, 3);
     expect(inputTypeLabel).toContain('Custom Input Type');
+
+    const placeHolderLabel = getHeadingText(wrapper, 4);
+    expect(placeHolderLabel).toContain('Custom Placeholder');
+  });
+
+  it('successfully populate placeholder input field', () => {
+    const div = document.createElement('div');
+    document.body.appendChild(div);
+
+    const wrapper = mount(<Card {...props} />, { attachTo: div });
+    const placeholder = wrapper
+      .find('.card-container')
+      .first()
+      .find('.card-text')
+      .at(6);
+    placeholder.simulate('change', { target: { value: 'New Placeholder' } });
```

Comment:
is this placeholder a distinct input/element? I'm asking because this looks like setting an input's value vs its native placeholder (see DOM API for native input placeholders here https://www.w3schools.com/jsref/prop_text_placeholder.asp) 🤔

Reply:
`placeholder` in this case is a property instead of a html element.